ctx.go

```diff
@@ -450,7 +451,49 @@ func (c *DefaultCtx) Cookie(cookie *Cookie) {
 // The returned value is only valid within the handler. Do not store any references.
 // Make copies or use the Immutable setting to use the value outside the Handler.
 func (c *DefaultCtx) Cookies(key string, defaultValue ...string) string {
-	return defaultString(c.app.getString(c.fasthttp.Request.Header.Cookie(key)), defaultValue)
+	value := c.app.getString(c.fasthttp.Request.Header.Cookie(key))
+	// If the value looks like binary data, return it as-is
+	if len(value) > 0 && !utf8.ValidString(value) {
+		return value
+	}
+	return defaultString(c.sanitizeCookieValue(value), defaultValue)
+}
+
+// sanitizeCookieValue sanitizes a cookie value according to RFC 6265.
+// It removes invalid characters from the cookie value, similar to how
+// Go's standard library handles cookie values.
+func (c *DefaultCtx) sanitizeCookieValue(v string) string {
```

Comment:
@gofiber/maintainers Is the correct approach to "clean" the cookie for the user. Or should we return HTTP 400?

@c00kie17 how does net/http handle this?